library/core/src/slice/iter.rs

```diff
@@ -355,6 +355,11 @@ impl<'a, T: 'a, P: FnMut(&T) -> bool> Split<'a, T, P> {
     pub(super) fn new(slice: &'a [T], pred: P) -> Self {
         Self { v: slice, pred, finished: false }
     }
+    /// Returns items which were not yet returned by split.
```

Comment:
The comment needs to be expanded. The tests below should be moved to doctests so users can see how it can be used.